tools/sweeper_resolver/main.go

```diff
@@ -0,0 +1,293 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/magic-modules/mmv1/api"
+	"github.com/GoogleCloudPlatform/magic-modules/tools/test-reader/reader"
+	"gopkg.in/yaml.v2"
+)
+
+// Custom logger that only prints when debug mode is enabled
+type debugLogger struct {
+	debug bool
+}
+
+func (l *debugLogger) Printf(format string, v ...interface{}) {
+	if l.debug {
+		log.Printf(format, v...)
+	}
+}
+
+type MetaData struct {
+	Resource       string `yaml:"resource"`
+	GenerationType string `yaml:"generation_type"`
+	SourceFile     string `yaml:"source_file,omitempty"`
+}
+
+func main() {
```

Comment:
We should run this using cobra to match our other tooling and because it's more readable and abstracts away some of the weird stuff about golang CLI implementation. It should return the basic data about what issues were detected, which would get used by the CI to generate inline comments (ideally using some kind of generic implementation that could handle issue creation for missing test detector and sweepable prefixes as well - maybe also for other arbitrary feedback we might want to create in the future. For example, we could return an object that contains a message plus the line number and file to attach it to. Any multi-line message should most likely be built using a go template.)